### PR TITLE
#180 add appveyor CI

### PR DESCRIPTION
**Pull request by Dekken**
initial commit for appveyor conf - to follow with fix for windows

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+branches:
+  only:
+    - master
+
+max_jobs: 1
+
+image:
+- Visual Studio 2015
+- Visual Studio 2017
```
Comment (Mbompr) [inline appveyor.yml]: So you build with 2 versions of visual studio? 
[Here](https://cowboyprogrammer.org/2014/06/building-python-wheels-for-windows/) they say that 2010 is the most important to be compatible with. But is this feasible with C++11?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+
+version: 0.5.{build}
```
Comment (Mbompr) [inline appveyor.yml]: Is this appveyor version?

Comment (Dekken) [reply appveyor.yml]: it is, but it's arbitrary - it's just for discerning "build numbers" o appveyor

File: appveyor.yml (hunk 1 above)
Comment (Dekken) [reply appveyor.yml]: that article is almost four years old, but there is a VS 2013 image, so we can do all three

Comment (Dekken) [reply appveyor.yml]: dist utils complains requiring MSVC > 2014 - so just the two it seems

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  ## Use Chocolatey to install SWIG.
+  # Only install swig if it isn't present (as a result of AppVeyor's caching).
+  # SWIG 3.0.8 is the minimum required version, but it does not yet exist in
+  # Chocolatey.
+  - IF NOT EXIST C:\ProgramData\chocolatey\bin\swig.exe choco install swig --version 3.0.9 --yes --limit-output #> $null
+
+build_script:
+  - git submodule update --init
+  - git clone https://github.com/X-DataInitiative/tick_appveyor -b master --depth 1 appveyor
+
```
Comment (Mbompr) [inline appveyor.yml]: # Python 3.6

File: appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  - C:\Python36-x64\Scripts\pip install appveyor\pip\numpy-1.14.0+mkl-cp36-cp36m-win_amd64.whl
+  - C:\Python36-x64\Scripts\pip install appveyor\pip\numpydoc-0.7.0-py2.py3-none-any.whl
+  - C:\Python36-x64\Scripts\pip install appveyor\pip\scipy-1.0.0-cp36-cp36m-win_amd64.whl
+  - C:\Python36-x64\Scripts\pip install -r requirements.txt
+  - C:\Python36-x64\Scripts\pip install tensorflow
+  - C:\Python36-x64\python setup.py build_ext --inplace 
+  - C:\Python36-x64\python -m unittest discover -v . "*_test.py"
+  - C:\Python36-x64\python setup.py bdist_wheel
+  - rm -rf build
+  - cat appveyor/pip/numpy/numpy-1.14.0+mkl-cp35* > appveyor/pip/numpy-1.14.0+mkl-cp35-cp35m-win_amd64.whl
```
Comment (Mbompr) [inline appveyor.yml]: # Python 3.5

File: appveyor.yml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  - C:\Python35-x64\Scripts\pip install appveyor\pip\numpy-1.14.0+mkl-cp35-cp35m-win_amd64.whl
+  - C:\Python35-x64\Scripts\pip install appveyor\pip\numpydoc-0.7.0-py2.py3-none-any.whl
+  - C:\Python35-x64\Scripts\pip install appveyor\pip\scipy-1.0.0-cp35-cp35m-win_amd64.whl
+  - C:\Python35-x64\Scripts\pip install -r requirements.txt
+  - C:\Python35-x64\Scripts\pip install tensorflow
+  - C:\Python35-x64\python setup.py build_ext --inplace 
+  - C:\Python35-x64\python -m unittest discover -v . "*_test.py"
+  - C:\Python35-x64\python setup.py bdist_wheel
+  - rm -rf build
+
```
Comment (Mbompr) [inline appveyor.yml]: # Python 3.4 : Building wheels for pandas is currently failing, having trouble with pandas

**Pull request merged**